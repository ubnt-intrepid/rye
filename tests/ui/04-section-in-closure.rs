fn main() {}

#[rye::test_case]
fn section_in_closure() {
    std::convert::identity(|| {
        section!("section", {
            assert!(true);
        });
    });
}
