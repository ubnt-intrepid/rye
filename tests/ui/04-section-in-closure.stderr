error: section cannot be described in a closure
 --> $DIR/04-section-in-closure.rs:6:9
  |
6 | /         section!("section", {
7 | |             assert!(true);
8 | |         });
  | |__________^
